### PR TITLE
Create new envelopes in DB for duplicates

### PR DESCRIPTION
**Pull request by rkondratowicz**
https://tools.hmcts.net/jira/browse/BPS-1065

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandler.java [hunk 1]
```diff
@@ -46,9 +45,14 @@ public void handle() {
                                 container,
                                 duplicate.fileName
                             );
+                            var id = envelopeService.createNewEnvelope(
+                                duplicate.container,
+                                duplicate.fileName,
+                                duplicate.blobCreatedAt
+                            );
 
+                            envelopeService.markAsRejected(id, "Duplicate");
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandler.java]: any ideas for rejection message?

Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandler.java]: perhaps can inclue reference to original if possible?

File: src/test/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandlerTest.java [hunk 2]
```diff
@@ -50,9 +49,12 @@ void should_check_all_containers() {
     }
 
     @Test
-    void should_move_blob_and_store_an_event() {
+    void should_move_blob__to_rejected_container_and_create_a_new_envelope() {
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandlerTest.java]: extra underscore :)

File: src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandler.java (hunk 1 above)
Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandler.java]: Duplicate envelope with zip filename {} for jurisdiction {} ?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandler.java]: > Duplicate envelope with zip filename {} for jurisdiction {} ?

event has a foreign key to envelope which has filename and jurisdiction so this is not adding any extra info.

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandler.java]: So just Duplicate envelope then ?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/DuplicateFileHandler.java]: done. Extracted to constant and updated tests

**Pull request merged**